wazuh/wazuh-dashboard-plugins
# Add create report definition

### Description

Adds the generation of dashboard report definitions when Wazuh is started.
 
### Issues Resolved

- #7087 

### Evidence

https://github.com/user-attachments/assets/f8780aa7-191a-45fc-97ad-67b9c0c0041a

#### Prerequisites

Have an environment with the wazuh dashboard reporting

### Test

1. Add sample data from vulnerability module
2. Navigate to the vulnerability module and enter the Dashboard by reference tab.
3. The first is to create the dashboard and the definition of the dashboard report.
4. Navigate to `Explore>Reporting`
5. It should have been created the definition of the Vulnerability Report.
6. Edit definition report
7. Navigate to the vulnerability module and enter the Dashboard by reference tab.
8. Click on Restart report definition
9. Check that the report definition returns to the default version.




### Check List
